.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't think we should be integrating the .project file...

Reply:
Sorry, not that one again. Second time this happens.